src/pages/basic_data/institution_admin/FindPersonPanel.tsx

```diff
@@ -81,6 +88,28 @@ export const FindPersonPanel = () => {
                   />
                 )}
               </Field>
+
+              <Field name="user.nvi">
+                {({ field }: FieldProps<NviVerification>) => (
+                  <Box sx={{ display: 'flex', alignItems: 'center' }}>
+                    <Checkbox
+                      {...field}
+                      value={values.user.nvi}
+                      onChange={() => {
+                        setConfirmedIdentity(!confirmedIdentity);
+                        setFieldValue(field.name, {
+                          verifiedBy: { id: !confirmedIdentity ? userCristinId : '' },
+                          verifiedAt: { id: !confirmedIdentity ? userTopLevelOrg : '' },
+                        });
+                        setFieldValue('user.nationalId', '');
+                      }}
+                    />
+                    <Typography sx={{ whiteSpace: 'pre-line' }}>
```

Comment:
Husk å sjekk a11y, her vil ikke skjermleser vite at dette er en label for checkbox. Label skal også være klikkbar for checkboxen.
Du kan se hvordan vi har wappet `Checkbox` inni `FormControlLabel` andre plasser